Automattic/jetpack
# Social: Hide "Share post" button when there are no connections

> we should hide the 'Share post' button when there are no connections


## Proposed changes:

* Hide "Share post" button in the editor when there are no connections

### Other information:


## Jetpack product discussion

> 

## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to post editor for a post that is already published
* Open Jetpack/Social sidebar
* Remove all the social connections you might have
* Confirm that you don't see the "Share post" button in "Share this post" section.

<table>
    <tbody>
        <tr>
            <th>BEFORE</th>
            <th>AFTER</th>
        </tr>
        <tr>
            <td>
<img width="284" alt="Screenshot 2024-06-26 at 1 53 22 PM" src="https://github.com/Automattic/jetpack/assets/18226415/6cd21dc8-4d79-4b05-aa17-0646b305b438">
</td>
            <td>
<img width="283" alt="Screenshot 2024-06-26 at 1 52 50 PM" src="https://github.com/Automattic/jetpack/assets/18226415/d8c278a5-e2aa-438d-ad11-fe8c44024ed6">
</td>
        </tr>
    </tbody>
</table>
